x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -1159,202 +1164,190 @@ describe('Task Runner', () => {
     `);
   });
 
-  testAgainstEphemeralSupport(
-    'actionsPlugin.execute is called when notifyWhen=onActionGroupChange and alert alert state has changed',
-    (
-        customTaskRunnerFactoryInitializerParams: TaskRunnerFactoryInitializerParamsType,
-        enqueueFunction: (options: ExecuteOptions) => Promise<void | RunNowResult>
-      ) =>
-      async () => {
-        customTaskRunnerFactoryInitializerParams.actionsPlugin.isActionTypeEnabled.mockReturnValue(
-          true
-        );
-        customTaskRunnerFactoryInitializerParams.actionsPlugin.isActionExecutable.mockReturnValue(
-          true
-        );
-        ruleType.executor.mockImplementation(
-          async ({
-            services: executorServices,
-          }: AlertExecutorOptions<
-            AlertTypeParams,
-            AlertTypeState,
-            AlertInstanceState,
-            AlertInstanceContext,
-            string
-          >) => {
-            executorServices.alertInstanceFactory('1').scheduleActions('default');
-          }
-        );
-        const taskRunner = new TaskRunner(
-          ruleType,
-          {
-            ...mockedTaskInstance,
-            state: {
-              ...mockedTaskInstance.state,
-              alertInstances: {
-                '1': {
-                  meta: {
-                    lastScheduledActions: { group: 'newGroup', date: new Date().toISOString() },
-                  },
-                  state: { bar: false },
+  test.each(ephemeralTestParams)(
+    'actionsPlugin.execute is called when notifyWhen=onActionGroupChange and alert alert state has changed %s',
+    async (nameExtension, customTaskRunnerFactoryInitializerParams, enqueueFunction) => {
```

Comment:
Suggest adding an `expect` to this test to ensure the correct number of triggered actions is logged to the event log.